pimd/pim_cmd.c

```diff
@@ -8406,11 +8427,19 @@ DEFUN_HIDDEN (interface_no_ip_pim_ssm,
 	      IFACE_PIM_STR)
 {
 	const struct lyd_node *igmp_enable_dnode;
-	char igmp_if_xpath[XPATH_MAXLEN + 20];
+	char igmp_if_xpath[XPATH_MAXLEN];
+
+	int printed =
+		snprintf(igmp_if_xpath, sizeof(igmp_if_xpath),
+			 "%s/frr-gmp:gmp/address-family[address-family='%s']",
+			 VTY_CURR_XPATH, "frr-routing:ipv4");
+
+	if (printed >= (int)(sizeof(igmp_if_xpath)) {
```

Comment:
Missed closed brace. Please rectify this.
if (printed >= (int)(sizeof(igmp_if_xpath))) {